.flake8

```diff
@@ -1,5 +1,6 @@
 [flake8]
 ignore = C901, E203, E266, E501, E731, W503
-max-line-length = 88
-max-complexity = 18
-select = B,C,E,F,W,T4
\ No newline at end of file
+select = B,C,E,F,W,T4
+per-file-ignores =
+    tests/*: E722, F811
+    dash_table_base/__init__.py: F401, F403
```

Comment:
I'm fine with different rules for tests vs source files, but putting single-file overrides in the config feels confusing to me, as well as potentially overly broad  - can we just override these rules on the specific lines that need them?

Reply:
Sure, my thought was that by making it part of the configuration it would be harder to sneak an exception in. I'm fine either way. Updating.